### PR TITLE
[FEATURE][HTML] Implement 'Rename' and 'Delete' operations for contacts

### PR DESCRIPTION
**Pull request by jafey**
https://saros-build.imp.fu-berlin.de/gerrit/#/c/3377/

### CODE REVIEW

File: de.fu_berlin.inf.dpp.ui.frontend/html/src/components/MainView/index.jsx [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+  onClickContactAction = e => {
+    const { jid } = e.target.parentNode.parentNode.dataset
+    const { action } = e.target.parentNode.dataset
+    if (action === 'rename') {
+      this.props.mainUI.doShowRenameContactView(jid)
+    } else if (action === 'delete') {
+      this.props.core.doDeleteContact(jid)
+    }
+  }
+
```
Comment (m273d15) [inline de.fu_berlin.inf.dpp.ui.frontend/html/src/components/MainView/index.jsx]: Remove superfluous arguments? displayName, ...

**Comment by m273d15** [comment]
I close this PR, because the whole functionality has to be reworked and the PR is stale. The PR is referenced in the corresponding issue #245.

**Comment by tobous** [comment]
@m273d15 Can the branch be deleted?

**Comment by m273d15** [comment]
@tobous Yes (I deleted the branch)

**Pull request closed without merging**